src/terragraph-e2e/e2e/minion/BgpUtils.cpp

```diff
@@ -172,67 +165,62 @@ BgpUtils::fetchVtyshBgpStatus() {
   }
 
   std::string bgpSummary = bgpSummaryOutput.value();
-  std::vector<std::string> summaryHeaders, summaryEntries;
-  int neighborCount = parseVtyshBgpTable(
-      bgpSummary, "Neighbor", summaryHeaders, summaryEntries);
 
-  // Iterate through neighbors and create BgpInfo per neighbor
-  for (int i = 0; i < neighborCount; ++i) {
-    // BGP Headers:
-    // Neighbor V AS MsgRcvd MsgSent TblVer InQ OutQ Up/Down State/PfxRcd
+ // dynamic object for  bgpSummary
+ folly::dynamic bgpSummaryInfo = folly::dynamic::object;
+
+ bgpSummaryInfo = folly::parseJson(bgpSummary)["peers"];
```

Comment:
You need to handle errors in `parseJson()` or else this code will crash.

Reply:
Addressed the review comments